cmd/project_example/Gopkg.toml

```diff
@@ -0,0 +1,12 @@
+[[constraint]]
+  name = "github.com/cryptopay-dev/yaga"
+  version = "1.8"
+
+[[constraint]]
+  name = "gopkg.in/go-playground/validator.v9"
```

Comment:
for what?... it is already in the Yaga :)

Reply:
https://github.com/cryptopay-dev/yaga/blob/bce5c14b703b19912163d4335d6cd571be9834bf/Gopkg.lock#L238-L242